crates/starknet-server/tests/get_transaction_by_hash.rs

```diff
@@ -45,11 +51,30 @@ mod get_transaction_by_hash_integration_tests {
         let declare_txn_v1: BroadcastedDeclareTransactionV1 =
```

Comment:
Aren't we now just using the class and not the whole tx? Perhaps no need to load the whole tx then? I would also say to then consider removing `declare_v1.json`, but I see it's used elsewhere.

Reply:
I also want to remove this file but for the same reason I did not

Reply:
[d97b751](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/213/commits/d97b751c17795db8afb95d00b72b5efc864b9248)

Reply:
I thought that somewhere we have just this class artifact (not wrapped in a declaration transaction). But if we don't, this is ok.